### ISSUE TITLE
Vignette does not load up

### ISSUE THREAD

**Issue by armish** [nickytong/drexplorer#2]
Here is why I get when I try to open the vignette:

```
R version 3.1.2 (2014-10-31) -- "Pumpkin Helmet"
Copyright (C) 2014 The R Foundation for Statistical Computing
Platform: x86_64-apple-darwin10.8.0 (64-bit)

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

R> library("drexplorer")
R> vignette("drexplorer")
Warning message:
vignette ‘drexplorer’ not found 
```

**Comment by nickytong** [comment]
Hi,
Thanks for using drexplorer. I reproduced your error on a Linux machine. For non-root installation, vignettes cannot be correctly pulled out for current R version (>R 3.0). There should be no problem if you install R package with root privilege or if you install it on Windows. I also found this to be true for other packages. I would recommend you to first check the vignette at: http://htmlpreview.github.io/?https://github.com/nickytong/drexplorer/blob/master/inst/doc/drexplorer.html until an ultimate solution is available.
